src/ml_tooling/baseclass.py

```diff
@@ -408,7 +409,10 @@ def score_estimator(
         logger.info("Scoring estimator...")
 
         if not data.has_validation_set:
-            raise DatasetError("Must run create_train_test first!")
+            data.create_train_test(
+                stratify=True if self.is_classifier else False,
+                seed=self.config.RANDOM_STATE,
```

Comment:
We should add the other configuration options here as as well.

Reply:
They have been added in the newest commit